packages/untp-test-suite/package.json

```diff
@@ -21,7 +21,7 @@
   },
   "scripts": {
     "untp": "node build/interfaces/cli/cli.js",
-    "test": "jest --config ./jest.config.js",
```

Comment:
If we do not call jest.config.js, should we delete this file?

Reply:
By default, Jest looks for a configuration file named "jest.config.js|ts|mjs|cjs|json" in the project's root directory. However, if you have a different configuration file or if you want to organize your configuration files differently, you can use the "--config" flag to specify the exact path to the configuration file you want Jest to use.

This can be useful in scenarios where you have multiple Jest configurations for different environments or scenarios, and you want to specify which one to use when running the tests.

Source: https://jestjs.io/docs/29.4/configuration